drivers/gles3/rasterizer_gles3.cpp

```diff
@@ -112,14 +112,22 @@ void RasterizerGLES3::begin_frame(double frame_step) {
 	//scene->iteration();
 }
 
-void RasterizerGLES3::end_frame(bool p_swap_buffers) {
+void RasterizerGLES3::end_frame(bool p_swap_buffers, bool p_sequential_sync) {
 	GLES3::Utilities *utils = GLES3::Utilities::get_singleton();
 	utils->capture_timestamps_end();
 }
 
-void RasterizerGLES3::gl_end_frame(bool p_swap_buffers) {
+void RasterizerGLES3::gl_end_frame(bool p_swap_buffers, bool p_sequential_sync) {
+	static const GLuint64 CLIENT_WAIT_SYNC_TIMEOUT = 1000000000; // One second
```

Comment:
Use number separators to make large numbers more readable:

```suggestion
	static const GLuint64 CLIENT_WAIT_SYNC_TIMEOUT = 1'000'000'000; // One second
```